browser/src/App.js

```diff
@@ -240,7 +240,7 @@ function App() {
           The team at Astoria Digital is using the data from these request to
           create visualizations to show their process in real time.
         </p>
-        <h2>Links and References</h2>
+        <h2 id="links_and_references">Links and References</h2>
         <ul>
           <li>All Requests</li>
```

Comment:
Could you also add [All Requests](https://www.muckrock.com/foi/list/?projects=778) and [Materials and Articles ](https://www.muckrock.com/project/new-york-officer-misconduct-and-disciplinary-materials-778/)

Reply:
I now realize that my wording was not clear. The ul in All Requests and Links and References need to be hyperlinked, and they are the same links as the ones in the nav bar for reference!

Reply:
Let's create @aniagaska's asks as a secondary issue.